lib/solidus_dev_support/rake_tasks.rb

```diff
@@ -34,10 +34,18 @@ def install_test_app_task
       ::CLOBBER.include test_app_path
 
       namespace :extension do
+        # We need to go back to the gem root since the upstream
+        # extension:test_app changes the working directory to be the dummy app.
+        #
+        # TODO: Fix this upstream on solidus/core.
         task :test_app do
           Rake::Task['extension:test_app'].invoke
           cd root
         end
+
+        directory ENV['DUMMY_PATH'] do
```

Comment:
It's not clear what this does. Can you explain it in a comment/commit message (whichever's relevant)?

Reply:
@elia nevermind, figured it out after your explanation!